lib/datadog/appsec/contrib/devise/patcher.rb

```diff
@@ -23,16 +24,26 @@ def target_version
           end
 
           def patch
-            patch_authenticable_strategy
+            patch_authenticatable_strategy
+            patch_rememberable_strategy
             patch_registration_controller
 
             Patcher.instance_variable_set(:@patched, true)
           end
 
-          def patch_authenticable_strategy
+          def patch_authenticatable_strategy
+            ::Devise::Strategies::Authenticatable.alias_method(:__validate, :validate)
```

Comment:
I suggest using a prefix/postfix that indicates the purpose and owner of this alias, e.g. something along the lines of `__validate_datadog` or `__validate_datadog_authenticatable`. With just a double-underscore prefix there is potential for collisions with other code and it's not clear what the purpose of the method is to someone who might be looking at it in their application for whatever reason (and has no knowledge of dd-trace-rb internals).